content/index.ja.md

```diff
@@ -0,0 +1,700 @@
+
+-   見出し、リスト、テーブル、入力フィールド、およびコンテンツ構造が適切にマークアップされている
+-   情報や指示の順序が、任意の表示から独立している
+-   ブラウザと支援技術が、表示をカスタマイズするための設定を提供している
+
+この要件を満たすことで、コンテンツが正しく音声で読み上げられたり、拡大されたり、さまざまな人々のニーズや好みに合わせて適応されたりすることが可能になります。例えば、読みやすさを向上させるために、カスタムの色組み合わせ、テキストサイズ、その他のスタイリングを使用して表示されることがあります。この要件は、ページのアウトラインや要約を自動生成するなど、他の形式の適応も容易にします。これにより、人々が概要を把握したり、特定の部分により簡単に焦点を当てたりするのに役立ちます。
+
+{% include excol.html type="start" %}
+
+#### 適応性に関するアクセシビリティ要件（技術仕様へのリンク）
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+**WCAG**
+
+-   [ガイドライン 1.3 - 適応可能](https://www.w3.org/WAI/WCAG22/quickref/#adaptable)
+
+**UAAG**
+
+-   [ガイドライン 1.4 - テキスト構成](https://www.w3.org/TR/UAAG20/#gl-text-config)
+-   [ガイドライン 1.5 - 音量構成](https://www.w3.org/TR/UAAG20/#gl-volume-config)
+-   [ガイドライン 1.6 - 合成音声の構成](https://www.w3.org/TR/UAAG20/#gl-speech-config)
+-   [ガイドライン 1.7 - ユーザースタイルシートの構成](https://www.w3.org/TR/UAAG20/#gl-style-sheets-config)
+-   [ガイドライン 1.9 - 代替ビュー](https://www.w3.org/TR/UAAG20/#gl-alternative-views)
+-   [ガイドライン 1.10 - 要素情報](https://www.w3.org/TR/UAAG20/#gl-info-link)
+
+**ATAG**
+
+-   [原則 A.1: オーサリングツールのユーザーインターフェイスは適用可能なアクセシビリティガイドラインに従う](https://www.w3.org/TR/ATAG20/#principle_a1)
+-   [ガイドライン A.2.2: (オーサリングツールのユーザーインターフェイス用) 編集ビューの表示がプログラムで決定できることを保証する](https://www.w3.org/TR/ATAG20/#gl_a22)
+-   [ガイドライン A.3.7: (オーサリングツールのユーザーインターフェイス用) プレビューが市場に出回っているユーザーエージェントと同等以上にアクセシブルであることを保証する](https://www.w3.org/TR/ATAG20/#gl_a37)
+-   [パート B. アクセシブルなコンテンツの制作をサポートする](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### 順応性に関連する記事
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee、色覚異常を持つオンラインショッパー](/people-use-web/user-stories/#shopper)
+-   [Alex、繰り返しストレス障害を持つ記者](/people-use-web/user-stories/#reporter)
+-   [Ilya、視覚障害を持つ上級職員](/people-use-web/user-stories/#accountant)
+-   [Preety、注意欠陥・多動性障害およびディスレクシアを持つ中学生](/people-use-web/user-stories/#classroomstudent)
+-   [Yun、低視力、手の震え、および軽度の短期記憶障害を持つ退職者](/people-use-web/user-stories/#retiree)
+-   [Luis、ダウン症のスーパーマーケットアシスタント](/people-use-web/user-stories/#supermarketassistant)
+-   [Kaseem、聴覚障害と視覚障害を持つ10代の若者](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+### コンテンツが見やすく、聞き取りやすくなる {#distinguishable}
+
+識別可能なコンテンツは見やすく、聞き取りやすい。そのようなコンテンツには以下のようなものがある：
+
+-   色が情報を伝えたりコンテンツを識別したりする唯一の方法として使用されていない
+-   デフォルトの前景色と背景色の組み合わせが十分なコントラストを提供している
+-   ユーザーがテキストを400%まで拡大したりテキストの間隔を変更したりしても、情報が失われない
+-   テキストが小さなウィンドウ（"ビューポート"）内で再配置され、ユーザーがテキストを大きくしたときも同様である
+-   テキストの画像がリサイズ可能であり、実際のテキストに置き換えられるか、可能な限り避けられる
+-    ユーザーがウェブサイトで再生される音声を一時停止、停止、または音量を調整できる
+-   背景音は低く設定されているか、干渉や気晴らしを避けるためにオフにできる
+
+この要件を満たすことは、前景と背景を区別し、重要な情報をより識別しやすくするのに役立ちます。これには、支援技術を使用しない人々や、背景の目立つ音声や視覚コンテンツから干渉を受ける可能性がある支援技術を使用する人々への配慮が含まれます。例えば、色覚異常の多くの人々は特定のツールを使用せず、テキストとその周囲の背景の間に十分な色のコントラストを提供する適切なデザインに頼っています。他の人にとっては、自動的に再生される音声がテキスト読み上げや[支援聴取デバイス (ALDs)](/teach-advocate/accessible-presentations/#ald "definition")と干渉する可能性があります。
```

Comment:
@takuan517 

Original version:
> This includes considerations for people who do not use assistive technologies and for people using assistive technologies who may observe interference from prominent audio or visual content in the background.

This part seems to have be reworded a little too much. Can you think of a formulation closer to the original version?

Same remark for:
> For instance, many people with color blindness do not use any particular tools and rely on a proper design that provides sufficient color contrast between text and its surrounding background

In particular, "provides sufficient color contrast between text and its surrounding background" seems to have been simplified into "provides adequate color contrast". It is important to not change or adapt or add or substract from the meaning in the original version.

Reply:
@remibetin 
fixed
https://github.com/w3c/wai-accessibility-principles/pull/50/commits/7aa74171bad5481aa206c5b67101b0879e82828d